macOS/DuckDuckGo/NewTabPage/Features/Customization/NewTabPageCustomizationModel.swift

```diff
@@ -32,35 +32,20 @@ final class NewTabPageCustomizationModel: ObservableObject {
         static let defaultColorPickerColor = NSColor.white
     }
 
-    struct DefaultBackgroundColorStyle {
-        let lightBackgroundColor: String
-        let darkBackgroundColor: String
-
-        init(theme: ThemeStyleProviding) {
-            let colors = theme.colorsProvider
-            lightBackgroundColor = colors.ntpLightBackgroundColor
-            darkBackgroundColor = colors.ntpDarkBackgroundColor
-        }
-    }
-
     let appearancePreferences: AppearancePreferences
     let customImagesManager: UserBackgroundImagesManaging?
     let sendPixel: (PixelKitEvent) -> Void
     let openFilePanel: () -> URL?
     let showAddImageFailedAlert: () -> Void
     let customizerOpener = NewTabPageCustomizerOpener()
 
-    @Published private(set) var backgroundColors: DefaultBackgroundColorStyle
```

Comment:
I've also noticed that `NewTabPageCustomizationProvider` has access to `AppearancePreferences`, which has a @Published property for the `themeName`.

This implies that we don't really need to observe + relay the Theme name here.

This change alone helped us drop quite some irrelevant code from Unit Tests. 🔥